indexer/services/comlink/src/controllers/api/v4/transfers-controller.ts

```diff
@@ -261,6 +235,59 @@ class TransfersController extends Controller {
       offset,
     };
   }
+
+  @Get('/compare')
+  async getTransferBetween(
+    @Query() sourceAddress: string,
+      @Query() sourceSubaccountNumber: number,
+      @Query() recipientAddress: string,
+      @Query() recipientSubaccountNumber: number,
+      @Query() createdBeforeOrAtHeight?: number,
+      @Query() createdBeforeOrAt?: IsoString,
+  ): Promise<TransferBetweenResponse> {
+    const sourceSubaccountId: string = SubaccountTable.uuid(sourceAddress, sourceSubaccountNumber);
+    const recipientSubaccountId: string = SubaccountTable.uuid(
+      recipientAddress,
+      recipientSubaccountNumber,
+    );
+
+    const [
+      transfers,
+      idToSubaccount,
+      idToAsset,
+      totalNetTransfers,
+    ]: [
+      TransferFromDatabase[],
+      SubaccountById,
+      AssetById,
+      string,
+    ] = await Promise.all([
+      TransferTable.findAll({
+        limit: config.API_LIMIT_V4,
+        senderSubaccountId: [sourceSubaccountId, recipientSubaccountId],
+        recipientSubaccountId: [sourceSubaccountId, recipientSubaccountId],
+        assetId: [USDC_ASSET_ID],
+        createdBeforeOrAt,
+        createdBeforeOrAtHeight: createdBeforeOrAtHeight
```

Comment:
nit: Seems a bit unintuitive that the transfers return are filtered, but the total returned is not. Was that considered when spec'ing out this endpoint?